tools/autograd/templates/Functions.cpp

```diff
@@ -665,6 +665,20 @@ Tensor var_backward(Tensor grad, const Tensor & self, IntArrayRef dim, bool unbi
   return (2.0 / (_safe_size(self.sizes(), dim) - unbiased)) * grad * (self - self.mean(dim, true));
 }
 
+Tensor var_std_mean_backward(const variable_list& grads, const Tensor & self, const Tensor & r1, const Tensor & r2, IntArrayRef dim, bool unbiased, bool keepdim, bool is_std) {
+  AT_CHECK(grads.size() > 0 && grads.size() <= 2, "grads size should be in interval [0,2]");
+  std::vector<Tensor> results;
+  if (grads[0].defined()) {
+    Tensor grad = is_std ? static_cast<Tensor>(grads[0]) / (r1 * 2) : static_cast<Tensor>(grads[0]);
+    results.push_back(var_backward(grad, self, dim, unbiased, keepdim));
+  }
+  if (grads[1].defined()) {
+    Tensor grad = static_cast<Tensor>(grads[1]);
```

Comment:
isn't this implicitly convertible?  So you can just do:
`Tensor grad = grads[1]`
?

Reply:
Fixed